страница match

adakhannn opened the pull request.

adakhannn committed:
страница match

katerinarodnykh commented on css/style.css:
```diff
@@ -844,3 +844,400 @@ table {
+
+.promo__information {
+  line-height: 22px;
+  color: var(--white);
```
весь текст в блоке за исключением группы белый, так что это свойство можно написать в .promo

katerinarodnykh commented on match.html:
```diff
@@ -0,0 +1,313 @@
+  			<p class="promo__text">Nike BoxMSK, ЦПКиО им. Горького, ул. Крымский Вал, 9</p>
+  		</div>
+  		<div class="promo__match">
+  			<div class="promo__team">
```
лишняя обёртка

katerinarodnykh commented on css/style.css:
```diff
@@ -844,3 +844,400 @@ table {
+}
+
+.promo__team {
+  width: 245px;
```
для таких моментов давай попробуем использовать для ширины функцию calc, прочитай про неё, с помощью неё можно задать ширину например calc((100% - ширина блока со счётом)/2)

katerinarodnykh commented on match.html:
```diff
@@ -0,0 +1,313 @@
+  				</a>
+  			</div>
+  			<div class="promo__score">
+  				<div class="promo__score-block">
```
блок со счётом внутри не разбивается на p, это просто текст 2 : 0, расстояние между цифрами не нужно совсем подгонять как под макет. Забыла сказать, что дизайнеры часто упускают некоторые важные моменты, например в счёте как мы помним из календаря может быть вывод счета по пенальти, учти его вывод, визуально он будет под счётом, меньше по размеру, в скобках (2:3), красного цвета, он может быть, а может не быть

katerinarodnykh commented on match.html:
```diff
@@ -0,0 +1,313 @@
+  					<p class="promo__colon">:</p>
+  					<p class="promo__number">0</p>
+  				</div>
+  				<div class="promo__tournament">
```
это не часть счёта, лучше сделать блоком под promo__match, делаем не абсолютом, возможно будет немного не по дизайну, но тут в турнирах может быть длинное название, поэтому лучше сделать так и сейчас там у тебя всё съехало https://prnt.sc/u7fm78, группа в дизайне  - жёлтым

katerinarodnykh commented on match.html:
```diff
@@ -0,0 +1,313 @@
+  					<img class="promo__image" src="img/second-team.png" alt="вторая команда" width="150" height="150">
+  				</a>
+  			</div>
+  			<nav class="promo__menu">
```
это отдельный блок, не часть promo, это внутренняя навигация, которая будет встречаться и на других страницах, можно назвать tabs, она не абсолютом. Мы делаем абсолютом только декоративные элементы или если у нас нет выбора (как например меню на слайдере), абсолют - это выравание элемента из общего потока в структуре, что не очень хорошо, если это не просто декаративный, а содержательный элемент.

katerinarodnykh commented on match.html:
```diff
@@ -0,0 +1,313 @@
+              <svg class="players__profile" width="30" height="30">
+                <use xlink:href="#profile"></use>
+              </svg>
+              Ибрагимов Линард
```
про таймлайн мы с тобой созвонимся, но некоторые вещи напишу, давай сразу учтёмнекоторые минимальные отступы между элементами, https://prnt.sc/u7fvj2  - сложно представить человека с таким длиннвм именем, но нам это нужно скорее на будущее, когда будем делать адаптив и ширина экрана будет уменьшаться

katerinarodnykh commented on match.html:
```diff
@@ -0,0 +1,313 @@
+  	  <div class="heading">
+        <h2 class="heading__text">судейский корпус</h2>
+      </div>
+      <div class="referee__wrapper">
```
это список

katerinarodnykh commented on match.html:
```diff
@@ -0,0 +1,313 @@
+  	<section class="statistic">
+  	  <div class="heading">
+        <h2 class="heading__text">Статистика матча</h2>
+      </div>
```
тут идёт список, где в каждом элементе списока текст и линия

katerinarodnykh commented on css/style.css:
```diff
@@ -844,3 +844,400 @@ table {
+  background-color: var(--yellow);
+}
+
+.promo__item:nth-child(2) {
```
не привязываемся к порядку, это активный элемент, к которому мы добавляем дополнительный класс модификатор --active, в html у тебя будет .promo__item--active

katerinarodnykh commented on css/style.css:
```diff
@@ -844,3 +844,400 @@ table {
+}
+
+.promo__link {
+  width: 140px;
```
предусматриваем длинное название на всякий случай https://prnt.sc/u7g5di, текст по центру, отступы по бокам

adakhannn committed:
правки match

не по дизайну , но сделал  как ты сказала , можно менять как угодно , каждая строка может быть , а можеть и не быть , перебрал все возможные варианты событий

adakhannn committed:
Update style.css

adakhannn committed:
правки match
